axelor-account/src/main/java/com/axelor/apps/account/service/moveline/MoveLineServiceImpl.java

```diff
@@ -464,4 +470,28 @@ protected List<MoveLine> getMoveLinesFromIds(List<Integer> moveLineIds) {
     }
     return moveLineList;
   }
+
+  @Override
+  public String computeMoveLineTaxLineSetDomain(int functionalOriginSelect, String dateString) {
+    String[] split = dateString.split("-");
+    LocalDate date =
+        LocalDate.of(
+            Integer.parseInt(split[0]), Integer.parseInt(split[1]), Integer.parseInt(split[2]));
```

Comment:
It would be better not to use a string value for the date. You can get the LocalDate value directly from the parentContext of the moveLine by using context.getParent().asType(Move.class), and move.getDate().

And if you want to parse a string date, it is better to use something like LocalDate.parse(dateString) or LocalDate.parse(dateString, DateTimeFormatter.ofPattern(pattern)) (with "pattern" being "yyyy-MM-dd" or any other pattern)

Reply:
This method is not used in this ticket, so there is no way to test it. I will put this modification in RM#83398.